mitodl/ol-data-platform
# refactor: Upgrade Healthchecks resource to use Pythonic interface

# What are the relevant tickets?
N/A

# Description (What does it do?)
Migrate the Healthchecks IO resource in Dagster to use their new Pydantic-based configuration interface.

# How can this be tested?
Launch the Dagster dev server targeting the open edX ops graph to ensure that the resources are loaded and configured.
